openqa-post-similarity

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+
+import os
+import Levenshtein
+from http import client
+from openqa_client.client import OpenQA_Client
+import time
+import json
+
+"""
+This script scans all the autoinst-log.txt in the testresults directory.
+Then find the error messages in the autoinst-log.txt.
+Use Levenshtein to calculate the text distance.
+Finally, post the results as comments by OpenQA_Client.
```

Comment:
I'd shorten and improve the wording a little bit.

```suggestion
This script scans all autoinst-log.txt files in the testresults directory searching
for error messages. Then it computes the Levenshtein distance and posts the
results as openQA comments.
```

Note that it would likely make more sense to show this text when the script is invoked via `--help` (and then of course anybody opening the file first can still read it as string literal).